dev/is_std_ptr.h

```diff
@@ -10,15 +10,42 @@ namespace sqlite_orm {
 
     template<typename T>
     struct is_std_ptr<std::shared_ptr<T>> : std::true_type {
+        using container_type = typename std::shared_ptr<T>;
+        using element_type = T;
+
         static std::shared_ptr<T> make(const T &v) {
             return std::make_shared<T>(v);
         }
+        static bool isEmpty(const std::shared_ptr<T> &v) {
+            return static_cast<bool>(v);
+        }
     };
 
     template<typename T>
     struct is_std_ptr<std::unique_ptr<T>> : std::true_type {
+        using container_type = typename std::unique_ptr<T>;
+        using element_type = T;
+
         static std::unique_ptr<T> make(const T &v) {
             return std::make_unique<T>(v);
         }
+        static bool isEmpty(const std::unique_ptr<T> &v) {
```

Comment:
`isEmpty` must be located in `type_is_nullable` as `operator()` not here. Please move it

Reply:
I'll just remove this method, I added it for my trial to remove code duplication in field_printer. I failed to make it compiles correctly. I should have removed it in my [rollback](https://github.com/fnc12/sqlite_orm/pull/418/commits/6d8b20c0442bf8912b7918fb13b3294cad7247bb), I forgot.